test/bin/manage_brew_rpms.sh

```diff
@@ -6,7 +6,7 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # its execution in a containerized environment with limited set of tools.
 
 usage() {
-    echo "Usage: $(basename "$0") [access | download <version> <path>]"
+    echo "Usage: $(basename "$0") [access | download <version> <path> <version_type>]"
```

Comment:
Version type seems to be optional.
```suggestion
    echo "Usage: $(basename "$0") [access | download <version> <path> [version_type]]"
```